packages/server/src/ui/routes/project-dashboard/graphs/category-score/category-score-graph.jsx

```diff
@@ -36,9 +36,17 @@ const VersionChangeWarning = () => {
   );
 };
 
-/** @param {{selectedBuildId: string|undefined, medianStatistics: Array<StatisticWithBuild>, versionChanges: Array<{build: LHCI.ServerCommand.Build}>, pinned: boolean}} props */
+const InfoTip = () => {
+  return (
+    <div className="category-score-graph__version-change-warning">
+      <i className="material-icons">info</i> Click to pin
```

Comment:
```suggestion
      <i className="material-icons">info</i> Click graph to pin
```

maybe? otherwise I could see people interpreting this as clicking *on the message*  to pin it or something

Reply:
good point. just was trying to be as concise as possible.

I was between this verbiage:

A) Click to pin/suspend
B) Click point to pin/suspend